frontend/ui/quickstart.lua

```diff
@@ -10,61 +10,185 @@ local lfs = require("libs/libkoreader-lfs")
+h1 { page-break-before: avoid; margin-bottom: 1em; }
+h2 { background-color: black; color: white; text-align: center; page-break-after: avoid; }
+
+hr { margin: 1em 20%; }
+div.generated { font-size: x-small; }
+
+li { margin: 0.5em 0; page-break-inside: avoid; }
+blockquote { text-align: center; page-break-before: avoid; page-break-inside: avoid; } /* Markdown lines starting with '>' */
+
+/* Inline image (icons) */
+p img, blockquote img { height: 1.5em; vertical-align: bottom; }
+
+/* Standalone image (UI screenshots) */
+div.img-block { text-align: center; margin: 0.5em; }
+div.img-block > img { max-height: 60vh; max-width: 80vw; }
+div.break-before-avoid { page-break-before: avoid; }
+div.break-after-avoid { page-break-after: avoid; }
+
+div.table { display: table; page-break-inside: avoid; width: 100%; }
+div.table > div { display: table-row; }
+div.table > div > * { display: table-cell; text-indent: 0; padding: 0.3em; }
+div.table > div > *:nth-child(2) { text-align: left; hyphens: none; background-color: #eeeeee; }
+div.table > div > *:nth-child(3) { white-space: nowrap; }
+]]
+
 local quickstart_guide = T(_([[
-# KOReader Quickstart Guide
+<div class="logo">![KOReader](../resources/koreader.svg)</div>
+
+# QUICKSTART GUIDE
+
+* [USER INTERFACE](#ui)
+* [USER INTERFACE TIPS](#uitips)
+* [ACCESSING FILES](#afiles)
+* [TRANSFERRING FILES](#tfiles)
+* [FRONTLIGHT/BACKLIGHT](#flight)
+* [WHILE READING](#reading)
+* [INSTALLING DICTIONARIES](#dicts)
+* [MORE INFO](#more)
+
+---
+
+You can access the complete user manual from [our GitHub page](https://github.com/koreader/koreader).
+
+## USER INTERFACE <a id="ui"></a>
+
+<div class="img-block">![Touch zones](../resources/quickstart/touchzones.png)</div>
+
+- To show the **TOP MENU** or **BOTTOM MENU** you can click the indicated zones. You can click or swipe down the upper zone to show the **TOP MENU**.
+
+- **STATUS BAR** zone can be used to cycle between STATUS BAR items if one item is visible. This will also hide and show the STATUS BAR if you tap enough times.
+
+## USER INTERFACE TIPS <a id="uitips"></a>
+
+- You can change interface language via:
+
+> **TOP MENU ➔ ![Settings](../resources/icons/mdlight/appbar.settings.svg) ➔ Language**
+
+- If you tap and hold an option or menu item (font weight, line spacing etc.), you can set its value as **DEFAULT**.  The new value will only be applied to documents opened from now on. Previously opened documents will keep their settings. You can identify default values as a STAR in menu or as a black border around indicators as seen below:
+
+<div class="img-block break-before-avoid">![Default setting 1](../resources/quickstart/defaultsetting1.png)</div>
+<div class="img-block break-before-avoid">![Default setting 2](../resources/quickstart/defaultsetting2.png)</div>
+
+- You can see explanations for all items on the **BOTTOM MENU** by tapping and holding the option’s name. This is also available for most of the **TOP MENU** menu items.
```

Comment:
These shouldn't be an UL or something?

```suggestion
- You can see explanations for all items on the **BOTTOM MENU** by tapping and holding the name of the option. This is also available for most of the **TOP MENU** menu items.
```

A phrase like "option's name" is generally reserved for a class of living things (people, animals, maybe plants).